### ISSUE TITLE
Don't ruin the challenge (until a while later)

### ISSUE THREAD

**Issue by TudbuT** [nop/antiantihuman#1]
Of course, I can't tell you what to do, but it would be nice to keep it challenging until a while later.

**Comment by DarkMetalMouse** [comment]
I agree, you should private this repository. By openly sharing the challenge solution you are preventing people from learning how to do it on their own.
